Lib/fontTools/varLib/__init__.py

```diff
@@ -722,8 +722,11 @@ def _add_COLR(font, model, master_fonts, axisTags, colr_layer_reuse=True):
 	if store:
 		mapping = store.optimize()
 		colr.VarStore = store
+		# don't add DeltaSetIndexMap for identity mapping
+		colr.VarIndexMap = None
 		varIdxes = [mapping[v] for v in merger.varIdxes]
-		colr.VarIndexMap = builder.buildDeltaSetIndexMap(varIdxes)
+		if any(i != varIdxes[i] for i in range(len(varIdxes))):
+			colr.VarIndexMap = builder.buildDeltaSetIndexMap(varIdxes)
```

Comment:
Thinking more, maybe we move that functionality into `buildDeltaSetIndexMap`, activated with a new parameter for backward compatibility, or default-enabled after auditing all places it is used.

Reply:
hm.. currently, all the varLib.builders functions return `self`, none of them ever returns `None`. I think it's fine like this